src/main.rs

```diff
@@ -111,6 +111,16 @@ fn execute(args: CliArguments) -> Result<FormatStatus> {
                     bail!("failed to format {error_count} files");
                 }
             }
+            #[cfg(feature = "completion")]
+            cli::Command::Completions { shell } => {
+                use clap::CommandFactory;
+                clap_complete::generate(
+                    *shell,
+                    &mut cli::CliArguments::command(),
+                    std::env::args().next().unwrap(),
+                    &mut std::io::stdout(),
+                );
+            }
         }
```

Comment:
do we need this? IIUC we already generate comp files in build script

Reply:
`cargo install` won't install asset files like completion, only the binaries. So we allow users to generate the completions themselves. Also, users can use the completions on the fly like:

```bash
source <(typstyle completion bash)
```

Reply:
what if we only keep the completion subcommand, and remove things build script🤔 i think this will make the code cleaner(we dont need cfgs)

Reply:
That sounds good, maybe I can revert some changes. I can generate completion scripts from the binary.